### ISSUE TITLE
Training is finished in a few seconds.

### ISSUE THREAD

**Issue by BlueAnthony** [AlexeyAB/darknet#822]
I'm trying to fine-tune yolov3 by KITTI object detection dataset with one GPU. 
When I start training yolov3, the training is just finished in few seconds. Even it doesn't show any error. 
But I think yolov3 doesn't be trained.
The result and the command is as below. 
![image](https://user-images.githubusercontent.com/17959032/40044985-a453f010-585b-11e8-87f4-02c9b7f9fdf0.png)

And there are some steps about my setting.
1. kitti.data
![image](https://user-images.githubusercontent.com/17959032/40043937-8741047a-5858-11e8-804c-afa1bb4bc6f7.png)

2.train.txt
![image](https://user-images.githubusercontent.com/17959032/40044001-b8405210-5858-11e8-8009-848bbc8e8473.png)
![image](https://user-images.githubusercontent.com/17959032/40044091-ef2f2ea4-5858-11e8-81f7-f6d102538a7a.png)

3.label.txt
![image](https://user-images.githubusercontent.com/17959032/40044189-3c7171ea-5859-11e8-815f-d62e79adac48.png)

4.kitti.names
![image](https://user-images.githubusercontent.com/17959032/40044225-5759769c-5859-11e8-8d9c-74b198a972b9.png)

5.yolov3_320_kitti.cfg
![image](https://user-images.githubusercontent.com/17959032/40044294-8845bafe-5859-11e8-9f74-df5e286b1001.png)
![image](https://user-images.githubusercontent.com/17959032/40044395-d1611d6e-5859-11e8-979e-f68adf554244.png)

Thanks for your helping.

**Comment by BlueAnthony** [comment]
I think I find the reason. The yolov3.weight will keep the max iteration in pre-trained phase.

**Comment by AlexeyAB** [comment]
Use `yolov3.conv.105` instead of `yolov3.weights` as pre-trained weights for training.

How to get `yolov3.conv.105`:
https://github.com/AlexeyAB/darknet/blob/8b5344ee2dc551dbe673020a33021e7f84f305f1/build/darknet/x64/partial.cmd#L21

**Comment by BlueAnthony** [comment]
@AlexeyAB What's the difference between yolov3.weights and yolov3.conv.105?
105 is the iteration?

**Comment by AlexeyAB** [comment]
105 is number of layers

**Comment by BlueAnthony** [comment]
@AlexeyAB  I mean the second 105 in "darknet.exe partial cfg/yolov3.cfg yolov3.weights yolov3.conv.105 105"

**Comment by AlexeyAB** [comment]
Yes, 105 is number of layers that you copy from yolov3.weights to yolov3.conv.105

**Comment by zrobotparking** [comment]
@BlueAnthony 
Sorry, but can you tell me how to convert KITTI into the format you used?

**Comment by bit-scientist** [comment]
@AlexeyAB where to download `yolov3.conv.105`?

**Comment by bit-scientist** [comment]
Hi, @AlexeyAB 
Let me ask you something. The reason behind `./darknet partial cfg/yolov3.cfg yolov3.weights yolov3.conv.105 105` is to make `max_batches` start from zero, right? Because, `yolov3.weights` was trained up to some batches and if we wanted to use `yolov3.weights`, we would have to find that last batch number and accordingly set our `max_batches` higher than that.
